doc/07_performance-tips.md

```diff
@@ -0,0 +1,16 @@
+# Performance Tips
+
+* Disable VSync and set threading to Single Safe in the project settings. Terrain building and editing performs significnatly faster with these settings.
+
+* Excessive noise height will kill your performance, use with care. A height range of 300 can run at 300-400fps on a GTX1060, but a range of 6-10,000 might run at 30-40fps. Hopefully this will be addressed when occlusion culling is available in Godot 4.
```

Comment:
It may also be caused by the fact invisible LODs are still included in Godot's frustrum culling, which is itself not very good compared to what's coming in 4.0.